base/src/main/java/io/spine/code/proto/Linker.java

```diff
@@ -62,13 +62,13 @@
     }
 
     static FileSet link(Collection<FileDescriptorProto> files) {
-        final Linker linker = new Linker(files);
+        Linker linker = new Linker(files);
         try {
             linker.resolve();
         } catch (DescriptorValidationException e) {
             throw newIllegalStateException(e, "Unable to link descriptor set files");
         }
-        final FileSet result = linker.getResolved()
+        FileSet result = linker.getResolved()
                                      .union(linker.getPartiallyResolved())
```

Comment:
The code formatting isn't quite adjusted automatically in cases like this one. Please review and address.